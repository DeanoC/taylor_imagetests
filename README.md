# taylor_imagetests
scripts and images to test the lua_image library works
