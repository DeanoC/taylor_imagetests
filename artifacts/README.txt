This directory holds the image generated by the test script. 
DO NOT COMMIT these artifacts
