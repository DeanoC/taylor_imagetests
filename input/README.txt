This folder contains various input file to test loaders etc.