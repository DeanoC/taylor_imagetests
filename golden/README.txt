The folder contains the blessed good version of the image produced by the test
Update when/if the results are deemed to have changed but are still valid
